source/index.html.md.erb

```diff
@@ -15852,48 +15897,70 @@ HTTP status code | Smartsheet errorCode | Smartsheet message
 
 # Formatting
 
-Formatting data can optionally be included for columns, rows, and cells by adding the "include=format" query string parameter to any API operation that returns any of those objects (for example, `GET /sheets/{sheetId}`, `GET sheets/{sheetId}/rows/{rowId}`, etc.).  When this parameter is included, objects that contain non-default formatting include a **format** property.  Objects that do not have any non-default format settings applied exclude this property.
- 
-Because the amount of format data in a large sheet can potentially be very large in itself, for bandwidth reasons, format data is represented by a compact format descriptor string.  The format descriptor takes the form of a comma-separated list of numeric values.  These values represent indexes into pre-defined format lookup tables which can be retrieved using the `GET /serverinfo` operation.
+> Example defaults value:
 
-## Row & Column Format
+```json
```

Comment:
This feels out of place in the right column - no anchored to any object

Reply:
I had trouble finding this--not sure if I found the right thing. The line numbers don't match up with the MD file line numbers. And a search got me nowhere. I *think* you were objecting to the json sample I had of the defaults value that was out of context, so I've removed that.